.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.formatOnSave": true,
+  "editor.codeActionsOnSave": {
+    "source.organizeImports": true,
+    "source.fixAll": true
+  },
+  "editor.wordWrap": "on"
+}
```

Comment:
Do we have a .gitignore file we can add this to? Committing this file will modify the IDE settings of anyone who opens the project in vs code